src/nautilus/paging.c

```diff
@@ -431,7 +431,7 @@ __fill_pd (pde_t * pd,
                 return;
             }
             memset(pt, 0, PAGE_SIZE_4KB);
-            pt[i] = (ulong_t)pt | flags;
+            pd[i] = (ulong_t)pt | flags;
```

Comment:
Yikes.  Definitely a bug - how could this have possibly worked, though?

Reply:
I'm also confounded. This definitely should have triggered, even on large page machines